src/models/transfer.py

```diff
@@ -218,3 +229,26 @@ def from_slippage(cls, slippage: SolverSlippage) -> Transfer:
             receiver=slippage.solver_address,
             amount_wei=slippage.amount_wei,
         )
+
+    @classmethod
+    def sort_list(cls, transfer_list: list[Transfer]) -> None:
+        """
+        This is the preferred and tested sort order we use for generating the transfer file.
+        It ensures that transfers are grouped
+        1. first by initial recipient (i.e. solvers),
+        2. then by token address (with native ETH last)
+        and finally order by amount descending (so that largest in category occur first).
+        Note that this method mutates the input data and nothing in returned.
+        """
+        transfer_list.sort(key=lambda t: (t.receiver, str(t.token), -t.amount))
```

Comment:
This is exactly the problem with this whole change here. If it were called solver then this wouldn't be much of a "Transfer" class anymore. If we wanted to do this properly, there should probably be some intermediary class which can become a transfer.

Reply:
Thinking about this a bit more... there is no real reason the class needs to be called `Transfer` since the only thing that matters in the end is the method `as_multisend_tx`. It could probably be named `SolverTransfer` and then we could rename `receiver` as `solver` and `redirect_to` as `reward_target` even.